src/System.IO.Compression/ref/4.0.0/System.IO.Compression.depproj

```diff
@@ -0,0 +1,13 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <AssemblyVersion>4.0.0.0</AssemblyVersion>
+    <OutputType>Library</OutputType>
+    <PackageTargetFramework>dotnet5.2</PackageTargetFramework>
```

Comment:
@ericstj, just verifying that this is correct as dotnet5.2 rather than dotnet5.4?  I see other changes in this commit going from dotnet5.2 to dotnet5.4, hence my question.